### PR TITLE
[14.0][ADD] apriory: Change name of edi_voxel_* modules to edi_voxel_*_oca

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

File: openupgrade_scripts/apriori.py [hunk 1]
```diff
@@ -38,6 +38,12 @@
     "edi_exchange_template": "edi_exchange_template_oca",
     "edi_storage": "edi_storage_oca",
     "edi_voxel": "edi_voxel_oca",
+    "edi_voxel_account_invoice": "edi_voxel_account_invoice_oca",
+    "edi_voxel_sale_order_import": "edi_voxel_sale_order_import_oca",
+    "edi_voxel_sale_secondary_unit": "edi_voxel_sale_secondary_unit_oca",
+    "edi_voxel_secondary_unit": "edi_voxel_secondary_unit_oca",
+    "edi_voxel_stock_picking": "edi_voxel_stock_picking_oca",
+    "edi_voxel_stock_picking_secondary_unit": "edi_voxel_stock_picking_secondary_unit_oca",
```
Comment (MiquelRForgeFlow) [inline openupgrade_scripts/apriori.py]:
```suggestion
    "edi_voxel_stock_picking_secondary_unit": "edi_voxel_stock_picking_secondary_unit_oca",  # noqa: B950
```

**Comment by pedrobaeza** [comment]
Not sure if they should be called `edi_voxel_oca_whatever...` instead. What do you think?

**Comment by sergio-teruel** [comment]
@pedrobaeza If you see the other modules of the repo, they follow the same pattern.
Yesterday I thought the same and I decided follow the same pattern ...

**Comment by sergio-teruel** [comment]
@MiquelRForgeFlow Changes done!!

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by etobella — APPROVED** [review]
Makes sense to me

**Pull request merged**